resotolib/resotolib/baseresources.py

```diff
@@ -758,6 +771,7 @@ class BaseNetworkShare(BaseResource, ABC):
 @define(eq=False, slots=False)
 class BaseVolume(BaseResource):
```

Comment:
Given that types don't mean anything in terms of cost and actual resources do cost money, making volume type and volume the same icon is confusing. I would make `*_type` something like a text/description icon as the type is just metadata and compute and storage some distinct icons as you actually pay for them.
I wonder if types need a distinct icon at all... they are mostly uninteresting and could have some generic default icon. I don't even think I would want them shown in most diagrams.